ArduPlane/AP_Arming.cpp

```diff
@@ -42,7 +42,7 @@ bool AP_Arming_Plane::pre_arm_checks(bool display_failure)
     }
     //are arming checks disabled?
     if (checks_to_perform == 0) {
-        return true;
+        return mandatory_checks(display_failure);
```

Comment:
As well as the new check we gain the base class checks in this case:

https://github.com/ArduPilot/ardupilot/blob/9cc8622ccc59cd2a64f06d79af53f91d8e4f36b1/libraries/AP_Arming/AP_Arming.cpp#L1589-L1598

Reply:
wont this prevent RC less operation? never advised but was not prohibited before....

Reply:
No, you just have to turn off the RC failsafe.